python/sklearnserver/sklearnserver/sklearn-multi-datatype-model.ipynb

```diff
@@ -0,0 +1,299 @@
+{
```

Comment:
This probably should be moved to sklearn v1 examples

Reply:
I moved.